### ISSUE TITLE
A change in batch size should trigger a change in gradient accumulation

### ISSUE THREAD

**Issue by saattrupdan** [saattrupdan/ScandEval#22]
To be able to make a change in batch size more reasonable when finetuning (different batch sizes tend to make the models perform differently), we should change the gradient accumulation accordingly.

Perhaps set a pre-specified list of possible batch sizes, like (1,2,4,8,16,32), and apply corresponding gradient accumulation (32,16,8,4,2,1).

**Comment by saattrupdan** [comment]
Implemented in https://github.com/saattrupdan/ScandEval/releases/tag/v0.2.0.
